src/main/java/frc/robot/commands/ArmElbowSetpointCommand.java

```diff
@@ -1,28 +1,23 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
 package frc.robot.commands;
-  
+
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import frc.robot.subsystems.Arm;
 
 // NOTE:  Consider using this command inline, rather than writing a subclass.  For more
 // information, see:
 // https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
 public class ArmElbowSetpointCommand extends InstantCommand {
```

Comment:
Like the comment says, we can easily replace this command with an inline command.

Here's an example of how to do so in `RobotContainer.java`:
```diff
-     xboxDpadUpButton.onTrue((new ArmShoulderSetpointCommand(65, m_arm))
+     xboxDpadUpButton.onTrue((new InstantCommand(() -> m_arm.setShoulderSetpoint(65), m_arm))
```

Reply:
I think moving to this inline method would be a good idea for all these simple `InstantCommand`s